### PR TITLE
Add first pass of unit tests for MDCBaseTextArea

### PR DESCRIPTION
**Pull request by andrewoverton**
This PR adds some initial unit tests to MDCBaseTextArea.

Related to #9407.

### CODE REVIEW

File: components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m [hunk 1]
```diff
@@ -0,0 +1,205 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
```
Comment (codeman7) [inline components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m]: Nit: 2020

File: components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+@end
+
+@interface MDCBaseTextAreaTests : XCTestCase
+@end
+
+@implementation MDCBaseTextAreaTests
+
+- (void)testFloatingLabelColorAccessors {
+  // Given
+  CGRect textAreaFrame = CGRectMake(0, 0, 130, 40);
```
Comment (codeman7) [inline components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m:31]: Can this be removed? Can't we just use the `alloc] init]`? Here and throughout.

File: components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+  MDCBaseTextArea *textArea = [[MDCBaseTextArea alloc] initWithFrame:textAreaFrame];
+  UIColor *floatingLabelColorNormal = [UIColor blueColor];
```
Comment (codeman7) [inline components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m]: I think as a convention we typically use the dot syntax. Here and throughout.

**Review by codeman7 — APPROVED** [review]
Overall looks good, I think we are pushing for test names to be `test<#input><#output>`. So `testFloatingLabelColorAccessors` would turn into `testFloatingLabelColorAccessorsSetProperties`

File: components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m (hunk 2 above)
Comment (andrewoverton) [reply components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m:31]: TextControls need to be told how wide to be. Initializing with a frame is one way to do that.

File: components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m (hunk 1 above)
Comment (andrewoverton) [reply components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m]: Done

File: components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m (hunk 3 above)
Comment (andrewoverton) [reply components/TextControls/tests/unit/BaseTextAreas/MDCBaseTextAreaTests.m]: Done

**Pull request merged**